dm/tests/dmctl_command/run.sh

```diff
@@ -552,6 +566,18 @@ function run_validator_cmd {
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"validation status test" \
 		"new\/ignored\/resolved: 0\/0\/0" 2
+	run_sql_source1 "insert into dmctl_command.t1 values(-10,'ignore-row')"
+	run_sql_source1 "create table dmctl_command.t_trigger_flush110(id int primary key)" # trigger flush
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
+		"\"synced\": true" 2
+	run_sql_tidb "select * from dmctl_command.t1"
+	check_not_contains "id: -10" # sync failed due to GO_FAILPOINTS
+	sleep 5                      # wait validator
```

Comment:
can this be removed too?

Reply:
Yes. Because validator is stopped, we don't need to wait for it. It's somehow a placeholder (pretending to wait for the validator) just in case there is something wrong to make stopping the validator failed. Perhaps just leave it here.

Reply:
> make stopping the validator failed

then we can just check whether there's `stage: stopped` in `validation status`